src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -47,6 +48,33 @@ public enum WebAuthenticationType
     }
 
     /// <summary>
+    /// The valid values for the -SslProtocol parameter for Invoke-RestMethod and Invoke-WebRequest
+    /// </summary>
+    [Flags]
+    public enum WebSslProtocol
```

Comment:
Is it possible to use this in other places? Then the name must be more generic.

Reply:
It's "public" in the way that other parameter enums are "public" but it shouldn't used in other places. users should use `SecurityProtocolType` or `SslProtocol` enums. This one is specific to the web cmdlets as `HttpClientHandler` doesn't support protocols in those Enums and thus the web cmdlets cannot consume them. I did this Enum to avoid having write yet another paramter logic set inside the cmdlet body.